app/experimenter/templates/experiments/section_results.html

```diff
@@ -0,0 +1,26 @@
+{% extends "experiments/section_base.html" %}
+
+{% block section_title %}
+Results
+{% endblock %}
+
+{% block section_content %}
+  <p><strong>Primary Result URL: </strong></p>
+  {% if experiment.results_url %}
+    <p>{{ experiment.results_url|linebreaks }}</p>
```

Comment:
would most likely want to `urlize` the experiments results_url
maybe also initial results and lessons learned if they're likely to include links too

Reply:
![giphy](https://user-images.githubusercontent.com/119884/62950948-11fd7980-bdb7-11e9-83b3-7dbb1da4fca2.gif)